release notes/v0.46.0.md

```diff
@@ -0,0 +1,139 @@
+- [#3181](https://github.com/grafana/k6/pull/3181) Adds a `RunOnEventLoop` method to the `modulestest.Runtime` type, which allows extensions developers to run code on the event loop from their tests.
+
+## Bug fixes
+
+- [#3178](https://github.com/grafana/k6/pull/3178), [xk6-grpc#23](https://github.com/grafana/xk6-grpc/pull/23) Fixes the registration of nested protobuf messages in the gRPC module. Thanks @thiagodpf for reporting and actually fixing the bug!
+- [#3194](https://github.com/grafana/k6/pull/3194) Fixes loki log output exiting before push is finished.
+- [#3231](https://github.com/grafana/k6/pull/3231) Fixes the tracing module sampling option to default to 1.0 when not set by the user.
+- [#3163](https://github.com/grafana/pull/3163) Addresses a bug in our gRPC example.
+- [#3196](https://github.com/grafana/k6/pull/3196) Logs packages lint fixes, and test for loki flushing at end.
+
+## Maintenance and internal improvements
+
+- [#3170](https://github.com/grafana/pull/3170) Upgraded the version of gRPC k6 demo service dependency for addressing a security vulnerability ([CVE-2023-32731]).
+- [#3211](https://github.com/grafana/k6/pull/3211) Updates the experimental gRPC module version used by k6.
+- [#3210](https://github.com/grafana/k6/pull/3210) Updates the xk6-output-prometheus-remote version used by k6.
+- [#3136](https://github.com/grafana/k6/pull/3136) Updates k6 Go dependencies.
+- [#3131](https://github.com/grafana/k6/pull/3131) Updates our Pull Request template to be more structured and include a checklist for contributors.
+- [#3177](https://github.com/grafana/k6/pull/3177) Updates the version of golangci-lint we use to the latest version.
+
+### Cloud output v2
+
+// TODO @codebien
+
+* [3104](https://github.com/grafana/k6/pull/3104) Retry and flushers pool
+* [3108](https://github.com/grafana/k6/pull/3108) Flush chunks
+* [3120](https://github.com/grafana/k6/pull/3120) Unlimited size for body payload
+* [3125](https://github.com/grafana/k6/pull/3125) Use a static remote service url
+* [3162](https://github.com/grafana/k6/pull/3162) Warn about labels that have reserved names
+* [3169](https://github.com/grafana/k6/pull/3169) Compact histogram
+- [#3182](https://github.com/grafana/k6/pull/3182) Fix off-by-one error for spans
+- [#3186](https://github.com/grafana/k6/pull/3186) Be less noisy when handling errors
+- [#3187](https://github.com/grafana/k6/pull/3187) Create batches for single bucket flush
+- [#3193](https://github.com/grafana/k6/pull/3193) Downscale the batch size
+- [#3195](https://github.com/grafana/k6/pull/3195) Make sure to use the overwritten config
+- [#3206](https://github.com/grafana/k6/pull/3206) Implement concurrent pushes accross batches
+- [#3226](https://github.com/grafana/k6/pull/3226) don't go over maxSeriesInBatch
```

Comment:
What is the expectation here? Should we include all the single bits? In https://github.com/grafana/k6/pull/3161 I just linked the main issue that contains the big parts. What is the value of reporting so many details?

Reply:
I share the feelings that we should not list so many details for the end user, especially for the things that are internal and are more like refactorings.

However, from the top of my head, I see one thing that is still better to stress. I mean 

```
* [3162](https://github.com/grafana/k6/pull/3162) Warn about labels that have reserved names
```

How about mentioning that the cloud output v2 drops labels that are reserved and right now it's `test_run_id` and all Prometheus system metrics (everything that prefixed with `__`).

What do you think?

Reply:
@olegbespalov good catch, I agree to stress on it

Reply:
I just have made it more clear indeed. I only listed all the issues to make sure they're reflected in the end result, and to support you with putting together your bit. Feel free to remove it as you see fit and replace it with your own content 👍🏻 🙇🏻